libs/core/algorithms/include/hpx/parallel/algorithms/find.hpp

```diff
@@ -549,8 +549,7 @@ namespace hpx { namespace parallel { inline namespace v1 {
                     util::loop_idx_n<std::decay_t<ExPolicy>>(base_idx, it,
                         part_size, tok,
                         [&f, &proj, &tok](type& v, std::size_t i) -> void {
-                            if (hpx::util::invoke(
-                                    f, hpx::util::invoke(proj, v)))
+                            if (hpx::util::invoke(f, HPX_INVOKE(proj, v)))
```

Comment:
```suggestion
                            if (HPX_INVOKE(f, HPX_INVOKE(proj, v)))
```

Reply:
Not sure why I missed this one... Will fix. Thanks!